Sage/carbon
# docs(textarea): ensure children arg is not set as an object FE-5125

fix #5046

### Proposed behaviour


Ensures `children` arg control is a string instead of an object and updates validation args to be the same

### Current behaviour


Arg (`children`, `error`, `warning` and `info`) controls are objects in `textarea-test.stories.js`

### Checklist




#### QA


### Additional context



### Testing instructions


`textarea-test.stories` - check the `children`, `error`, `warning` and `info` controls are string instead of object

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
